extensions/book/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "book",
+    "displayName": "book",
+    "description": "Support for Jupyter Books",
+    "version": "0.0.1",
+    "publisher": "Microsoft",
+    "engines": {
+        "vscode": "*",
+        "azdata": "*"
+    },
+    "categories": [
+        "Other"
+    ],
+    "activationEvents": [
+        "*"
+    ],
+    "main": "./out/extension",
+    "contributes": {
+        "views": {
+            "explorer": [
+                {
+                    "id": "bookTreeView",
+                    "name": "Books",
+                    "when": "bookOpened"
+                }
+            ]
+        },
+        "commands": [
+            {
+                "command": "bookTreeView.refreshEntry",
+                "title": "Refresh",
+                "icon": {
+                    "light": "resources/light/refresh.svg",
+                    "dark": "resources/dark/refresh.svg"
+                }
```

Comment:
It looks like we have a refresh_inverse.svg for dark theme - [https://github.com/microsoft/azuredatastudio/blob/master/extensions/azurecore/resources/dark/refresh_inverse.svg](https://github.com/microsoft/azuredatastudio/blob/master/extensions/azurecore/resources/dark/refresh_inverse.svg)